sound/soc/intel/boards/sof_sdw_common.h

```diff
@@ -56,15 +56,18 @@ enum {
 #define SOF_SDW_CODEC_TYPE_AMP		1
 #define SOF_SDW_CODEC_TYPE_MIC		2
 
+#define SOF_SDW_CODEC_MAX_DAI_NUM	3
```

Comment:
where does this come from?

Reply:
> where does this come from?

Sorry, I forgot to answer this commit. The `3` is because I assume that we have at most 3 different functions, like jack, amp, and mic dais in a codec. Ideally, that should be allocated dynamically according to `dai_num`, but I didn't find a way to do so. Thus, I define a reasonable maxim dai number instead.